Fitness/Views/Cells/DiscoverProsCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ProCell.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 4/22/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import Alamofire
+import AlamofireImage
+
+class DiscoverProsCell: UICollectionViewCell {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.discoverProsCell
+    var collectionView: UICollectionView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        //COLLECTION VIEW LAYOUT
```

Comment:
`space after //`

Reply:
I'll add the spaces in this files and others that don't have them